cf-templates/stack.yaml

```diff
@@ -66,15 +66,10 @@ Parameters:
     Type: String
     Default: "r6g.large.search"
   
-  MasterUserName:
+  OpenSearchMasterUserName:
     Description: Amazon OpenSearch Service - Username
     Default: "aosadmin"
     Type: String
-
-  MasterUserPassword:
-    Description: Amazon OpenSearch Service - Password
-    Default: "P@ssw0rd!"
-    Type: String
   
   # Reverse Proxy Variables
   ReverseProxyInstanceType:
```

Comment:
Did you test reverse proxy access after this change?

Reply:
Absolutely!